podvm/files/etc/systemd/system/netns@.service

```diff
@@ -0,0 +1,6 @@
+[Service]
```

Comment:
the file name is `netns@.service` it's a systemd template unit. so you "invoke" it when you require "netns@podns.service" somwhere. (this might be useful if we want to somehow paremeterize the podns at a later point, you could also just hardcode the name and call the file `podns-netns.service` or something)

NetworkNamespacePath should be a systemd-native way of executing binaries in a namespace.

Reply:
Thanks @mkulke